calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -106,22 +118,47 @@ public void DeleteLastInput()
 
         public void ShowRemainderOnDisplay(string remainder)
         {
-
+            AssignOperator(remainder);
         }
 
         public void ShowDecimalOnDisplay(string decimalDot)
         {
-
+            if (!hasDecimal)
+            {
+                DisplayContent += decimalDot;
+                hasDecimal = true;
+            }
         }
 
         public void Equals()
         {
-
+            switch (_operator)
+            {
```

Comment:
To reduce the repeated statement of `Convert.ToDouble(DisplayContent)`, you should assign a local variable to it here since it is converted in every case statement.